experimental/PropertyDDS/packages/property-properties/.eslintrc.cjs

```diff
@@ -42,6 +41,7 @@ module.exports = {
 		"@typescript-eslint/unbound-method": "off",
 		"guard-for-in": "off",
 		"import/no-duplicates": "off",
+		"import/no-extraneous-dependencies": "off",
```

Comment:
Add a comment to PR description about this one since it's a different case. See my PR description for something you can base it on.